### ISSUE TITLE
What to do with webgpu.org 

### ISSUE THREAD

**Issue by greggman** [gpuweb/gpuweb#4032]
We've discussed this in the meeting but I felt like it might be more productive to discuss it where we can write down ideas

For background, apparently the committee needs to decide what to do with `webgpu.org` (And `webgpu.io`?)

My position is, `webgpu.org` should not be run by this committee. Instead it should be given to a few people to be run as an "unofficial" website with a disclaimer that it's "not affiliated with the W3C GPU Committee"

# Why not be an "official" website?

Because it will take too much effort, friction, and argument to approve things. The committee has enough to do discussing the spec.

As an *official* site, at feels like anything someone wants to add will have to
be litigated. Examples:

* Should the site be allowed to link to Dawn or wgpu or Tint or Naga?
* Should the site be allowed to have articles or links about native WebGPU usage?
* Do examples need to be in JavaScript or are TypeScript, C++, rust, python, etc.. ok
* If there are docs, who's docs?
* If there are best practices, who decides what's "best"
* Is it ok if there are best practices that are different for different platforms or is such info against the spirit of interoperability?
* Which projects/libraries should be hosted or linked to?
   * Are they official projects/libraries of the committee or just happen to be hosted/linked
   * Which ones are listed first?
   * Are hosted libraries/projects blessed by the committee as official (this is the recommended solution for X)
* Are links to commercial products and services allowed?
* Can tangentially related things be linked/hosted/articles? For example the WebGPU and the `<model>` tag.

# Why not `webgpu.graphics` or `webgpu-info.org` or `realwebgpu.org` and leave the `webgpu.org` for "official" stuff?

Some concerns were raised that a site at `webgpu.org` will appear official even if there is a disclaimer "not affiliated with the W3C GPU Committee". Part of that issue is that many of the people likely working on content for the site happen to be on the committee.

I agree it's true it will appear official even with a disclaimer. But, would it be any different if those people made `real-webgpu.org`? If it's the place where there's activity and useful info, and if it's mostly run by people who happen to be on the W3C GPU Committee, then it seems like `real-webgpu.org` would be perceived as having an air of being "official". 

So, if the committee decides keep `webgpu.org` official and therefore of limited scope, and, as a result, the people go make `real-webgpu.org` or host all the content they were hoping to put at `webgpu.org`, none of the concerns will have been prevented. The committee will have no say over what happens at `real-webgpu.org` but, based on its membership, it will still be perceived as "official". 

In other words, the committee should give up webgpu.org as an official site.

**Comment by kainino0x** [comment]
For reference, the full list of things we (this group or google) have now, that I recall:

- github.com/gpuweb
- github.com/webgpu
- webgpu.org
- webgpu.dev
- webgpu.io
- webgpu.graphics

**Comment by frzi** [comment]
Unless the committee is planning to turn `webgpu.org` into something akin to [webassembly.org](https://webassembly.org/) — a small but central place with general information, compatibility tables, etc — and has the time and energy to keep it up to date, I'd rather see the domain in the hands of the community. This includes other domains as well.

Even [webgpu.dev](https://webgpu.dev), as handy as the shortcut is to the official W3C spec, would probably be better in the hands of the community. Anyone specifically looking for the spec can easily find it through other means (MDN or major search engines).

These domains could be useful hubs for WebGPU information maintained by enthusiastic people of the community; sharing links to blog posts, tutorials, frameworks, published books, etc. without having to go through a political checklist of 'should we or should we not?' I'm not confident a website maintained by a/the committee would have the same traction. Having these domains parked without any future plans seems detrimental in my opinion.

(`real-webgpu.org` sounds real sketchy too ;) )

**Comment by Kangz** [comment]
Closing in favor of discussing more on https://github.com/webgpu/webgpu.org
